mappings/net/minecraft/world/BlockCollisions.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/unmapped/C_dzzsobqg net/minecraft/world/BlockCollisions
```

Comment:
I don't feel like this should be part of the `world` package. Could be wrong though.

Reply:
I think look at it, it fits with `BlockLocating` which is also in the `world` package. I could also be wrong but i think it is fine